config/olm/kubernetes/bundle-0.2.1.Dockerfile

```diff
@@ -10,10 +10,6 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.5.0
```

Comment:
the latest operator sdk is 1.13.0, please double check that this value is correct